src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -50,12 +52,18 @@ public struct GCExtendedGamepadSnapShotDataV100 {
 		public float /* float_t = float */ LeftTrigger;
 		public float /* float_t = float */ RightTrigger;
 
-		// radar: https://trello.com/c/7FoGTORD (GCExtendedGamepadSnapShotDataV100 struct size / alignment not backward compatible)
-		// [TV (12, 1), Mac (10, 14, 1, onlyOn64: true), iOS (12, 1)]
-		// public bool LeftThumbstickButton;
-		// [TV (12, 1), Mac (10, 14, 1, onlyOn64: true), iOS (12, 1)]
-		// public bool RightThumbstickButton;
-
+		// Apple inserted these in the middle of the struct previously so it broke the struct for backwards compatibility,
+		// but now they are at the end of the struct so they are allowed and do not prevent backwards compatibility
+		[iOS (12,2), Mac (10,14,4), TV (12,2)]
+		public bool SupportsClickableThumbsticks;
```

Comment:
I looked at the header, and these fields are not in the `GCExtendedGamepadSnapShotDataV100` struct, they're only in the `GCExtendedGamepadSnapshotData` struct.

Reply:
Oh man big rookie mistake, thank you for catching this!